db/compaction/compaction_iterator_test.cc

```diff
@@ -968,6 +985,45 @@ TEST_F(CompactionIteratorWithSnapshotCheckerTest, CompactionFilter_FullMerge) {
       compaction_filter.get());
 }
 
+// Tests how CompactionIterator work together with AllowIngestBehind.
+class CompactionIteratorWithAllowIngestBehindTest
+    : public CompactionIteratorTest {
+ public:
+  bool AllowIngestBehind() const override { return true; }
```

Comment:
It seems more conventional to add an `allow_ingest_behind` parameter to `RunTest()` and `InitIterators()`. But we can take this either way.